source/_components/yeelight.markdown

```diff
@@ -166,6 +166,57 @@ Start flow with specified transitions
 
+### Service `yeelight.set_color_scene`
+
+Changes the light to the specified RGB color and brightness. If the light is off, it will be turned on.
+
+| Service data attribute    | Optional | Description                                                                                 |
+|---------------------------|----------|---------------------------------------------------------------------------------------------|
+| `entity_id`               |       no | Only act on a specific lights.                                                              |
+| `rgb_color`               |       no | A list containing three integers between 0 and 255 representing the RGB color you want the light to be. Three comma-separated integers that represent the color in RGB, within square brackets.|
+| `brightness`              |       no | The brightness value to set (1-100).                                                        |
+
+### Service `yeelight.set_hsv_scene`
+
+Changes the light to the specified HSV color and brightness. If the light is off, it will be turned on.
+
+| Service data attribute    | Optional | Description                                                                                 |
+|---------------------------|----------|---------------------------------------------------------------------------------------------|
+| `entity_id`               |       no | Only act on a specific lights.                                                              |
+| `hs_color`                |       no | A list containing two floats representing the hue and saturation of the color you want the light to be. Hue is scaled 0-360, and saturation is scaled 0-100.    |
+| `brightness`              |       no | The brightness value to set (1-100).                                                        |
+
+### Service `yeelight.set_color_temp_scene`
+
+Changes the light to the specified color temperature. If the light is off, it will be turned on.
+
+| Service data attribute    | Optional | Description                                                                                 |
+|---------------------------|----------|---------------------------------------------------------------------------------------------|
+| `entity_id`               |       no | Only act on a specific lights.                                                              |
+| `kelvin`                  |       no | Color temperature in Kelvin.                                                                |
+| `brightness`              |       no | The brightness value to set (1-100).                                                        |
+
+### Service `yeelight.set_color_flow_scene`
+
+Starts a color flow. Difference between this and [yeelight.start_flow](#service-yeelightstart_flow), this service call uses set_scene api. If light was off it will be turned on. There might be some firmware differences, in handling complex flows etc.
```

Comment:
You may want to add a link to this `set_scene` API.

Reply:
I've reworded this. I want'ed to indicate it uses other internal api call ( so that light will be turned on if it was on ). [Set_scene](https://gitlab.com/stavros/python-yeelight/blob/master/yeelight/main.py#L617) api is somehow confusing, when comes to arguments etc. Maybe its not necessary for user ?